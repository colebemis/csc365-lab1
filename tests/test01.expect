Enter query: LINHART, LELA, 3, 107, FAFARD, ROCIO
Enter query: 