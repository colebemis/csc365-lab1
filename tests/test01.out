Enter query: Querying student with last name LINHART
LINHART, LELA, 3, 107, FAFARD, ROCIO
Enter query: 